BlogTemplate/Pages/Post.cshtml

```diff
@@ -101,12 +102,12 @@
                     <form id="comment-form" method="post">
                         <div class="form-group">
                             <label for="AuthorName">Name</label>
-                            <input type="text" class="form-control" placeholder="" asp-for="NewComment.AuthorName" />
+                            <input type="text" class="form-control" placeholder="" maxlength="100" asp-for="NewComment.AuthorName" />

```

Comment:
maxlength should be enforced on the server side, e.g. with Data Annotations.

Reply:
This was fixed in newest commit